tests/test_registry.py

```diff
@@ -347,16 +392,16 @@ def factory(foo: svcs.Container):
 
         assert svcs._core._takes_container(factory)
 
-    def test_annotation_str(self):
-        """
-        Return true if the first argument is annotated as `svcs.Container`
-        using a string.
-        """
+    @pytest.mark.parametrize(
+        "module_source", takes_containers_annotation_string_modules
+    )
+    def test_annotation_str(self, module_source, create_module):
+        """Return `True` if the first argument is annotated as `svcs.Container`
+        using a string."""
```

Comment:
```suggestion
        """
        Return `True` if the first argument is annotated as `svcs.Container`
        using a string.
        """
```

Reply:
Aah sorry about that. I'd read it in the contributing guidelines, but I forgot to actually write it like that :P